### PR TITLE
feat(docs): Some SBOM policies

### LINKED ISSUE #1124: Create SBOM policies
Improve current and create new SBOM policies. The goal is to have a set of full examples oriented to SBOM quality metrics.

### PR DESCRIPTION
**Pull request by jiparis**
This improves and adds some new policies, including:
* SBOM is present
* all components have license
* no banned licenses
* no banned packages
* required packages
* SBOM freshness (not older than xxxx)

All of them will output an individual message for each violation (for example, the list of packages without licenses)

Closes #1124

### CODE REVIEW

File: docs/examples/policies/sbom/sbom-syft.yaml [hunk 1]
```diff
@@ -16,14 +16,15 @@ apiVersion: workflowcontract.chainloop.dev/v1
 kind: Policy
 metadata:
   name: made-with-syft
+  description: Verifies that the SPDX was created with Syft
+  annotations:
+    category: sbom
 spec:
   type: SBOM_SPDX_JSON
```
Comment (migmartri) [inline docs/examples/policies/sbom/sbom-syft.yaml:23]: this is specifically about spdx, should we call it `spdx-sbom-made-with-syft` and maybe add one for cycloinedx json?

File: docs/examples/policies/sbom/cyclonedx-freshness.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: workflowcontract.chainloop.dev/v1
+kind: Policy
+metadata:
+  name: cyclonedx-required-packages
+  description: Checks that SBOM has been generated recently
```
Comment (migmartri) [inline docs/examples/policies/sbom/cyclonedx-freshness.yaml:19]: how recent is this? Should we add it in the description until it's configurable?

File: docs/examples/policies/sbom/cyclonedx-banned-packages.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  annotations:
+    category: sbom
+spec:
+  type: SBOM_CYCLONEDX_JSON
+  embedded: |
+    package main
+    
+    import rego.v1
+    
+    banned_packages := ["log4j@2.14.1"]
```
Comment (migmartri) [inline docs/examples/policies/sbom/cyclonedx-banned-packages.yaml:29]: I'd add a comment explaining what's supported, with our without version, etc

**Review by migmartri — APPROVED** [review]
Awesome

**Comment by migmartri** [comment]
@jiparis any reason you are holding the merge? Maybe review from @javirln?

**Comment by jiparis** [comment]
> @jiparis any reason you are holding the merge? Maybe review from @javirln?

I'll add your suggestions and merge it right away.

**Pull request merged**